sdk/react/bottomOverlay.js

```diff
@@ -117,13 +117,20 @@ var BottomOverlay = React.createClass({
     var topOffset = this._renderTopOffset(scrubberSize);
     var leftOffset = this._renderLeftOffset(scrubberSize, percent, progressBarWidth);
     var positionStyle = {top:topOffset, left:leftOffset};
+    var scrubberStyle = this._customizeScrubber();
 
     return (
-      <View style={[styles.progressScrubber, positionStyle,{width:scrubberSize, height:scrubberSize}]}>
+      <View style={[scrubberStyle, positionStyle,{width:scrubberSize, height:scrubberSize}]}>
       </View>
       );
   },
 
+  _customizeScrubber: function() {
+    var scrubberStyle = {flex: 0, position: "absolute", backgroundColor: this.props.config.controlBar.scrubberBar.scrubberHandleColor, 
```

Comment:
Another idea is to set default color values if scrubberHandleColor and scrubberHandleBorderColor are not defined.